include/mvd/bits/sonata_misc.hpp

```diff
@@ -172,6 +172,12 @@ inline std::vector<std::string> SonataFile::getSynapseClass(const Range & range)
     return pop_->getAttribute<std::string>(did_synapse_class, select(range, size_));
 }
 
+inline std::vector<TSV::TSVInfo> SonataFile::getTSVInfo(const Range & range ) const{
+    // Avoid compiler warning: unused parameter 'range'
```

Comment:
Could you get away with just calling the function `getTSVInfo(const Range&)` (no parameter name)?